fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckState.kt

```diff
@@ -146,9 +148,28 @@ sealed interface ReviewQualityCheckState : State {
         data class AnalyzeLink(val url: String) : LinkType()
 
         /**
-         * Opens an external "Learn more" link.
+         * Opens an external link.
          */
-        data class ExternalLink(val url: String) : LinkType()
+        abstract class ExternalLink : LinkType() {
```

Comment:
Guess we are moving away from adding this in the state with #3717

Reply:
Rebased this on top of #3717